web/src/popover_menus.js

```diff
@@ -918,4 +931,268 @@ export function initialize() {
             popover_instances.send_later = undefined;
         },
     });
+
```

Comment:
It is hard to verify the changes below since they are being deleted and added in separate commits. It would be better to just have them in one commit.